modules/ROOT/pages/available-menu-items.adoc

```diff
@@ -37,6 +37,11 @@ include::partial$misc/plugin-menu-item-id-boilerplate.adoc[]
 :plugincode: advtable
 include::partial$misc/plugin-menu-item-id-boilerplate.adoc[]
 
+:plugincategory: premium
+:pluginname: Advanced Typography
+:plugincode: typography
+include::partial$misc/plugin-menu-item-id-boilerplate.adoc[]
```

Comment:
```suggestion
:pluginpage: advanced-typography.adoc
include::partial$misc/plugin-menu-item-id-boilerplate.adoc[]
```

Either use the above suggestion or rename `advanced-typopgrahy.adoc` to `typopgrahy.adoc`.